frontend/SplitPane.js

```diff
@@ -21,13 +21,18 @@ type Props = {
   left: () => React$Element,
   right: () => React$Element,
   initialWidth: number,
+  initialHeight: number,
+  isVertical: bool,
 };
 
-type DefaultProps = {};
+type DefaultProps = {
+  isVertical: true,
+};
 
 type State = {
   moving: boolean,
-  width: number,
+  width: string,
+  height: string,
```

Comment:
Thanks for pushing for this, I knew it was a bit of a smell when I was writing it, but my focus was on getting the `flow check` to pass.

Swapping the type back to a number has simplified the code and made it a lot less smelly